Source/OxyPlot.Pdf/PdfReportWriter.cs

```diff
@@ -139,6 +139,7 @@ public virtual void Close()
             r.RenderDocument();
             r.PdfDocument.Save(this.Output);
             this.CleanTemporaryPlotFiles();
+            this.Output.Close();
```

Comment:
if the ctor with stream is being used the close method of the stream can be called after the dispose of the pdfreportwriter. if the close method of the stream is being called before the dispose of the pdfreportwriter you get into trouble.